CovertInfiltration/Src/CovertInfiltration/Classes/X2Ability_SitRepAbilitySet_CI.uc

```diff
@@ -0,0 +1,169 @@
+	Template.Hostility = eHostility_Neutral;
+    Template.AbilityToHitCalc = default.DeadEye;
+    Template.AbilityTargetStyle = default.SelfTarget;
+    Template.AbilityTriggers.AddItem(default.UnitPostBeginPlayTrigger);
+    Template.bIsPassive = true;
+
+    StatEffect = new class'X2Effect_PersistentStatChange';
+    StatEffect.BuildPersistentEffect(1, true, false, true);
+    StatEffect.AddPersistentStatChange(eStat_Dodge, default.PHYSICAL_CONDITIONING_VALUE); //TODO: config this
+    StatEffect.SetDisplayInfo(ePerkBuff_Passive, default.PhysicalConditioningFriendlyName, default.PhysicalConditioningFriendlyDesc, Template.IconImage, true, ,Template.AbilitySourceName);
+    Template.AddTargetEffect(StatEffect);
+
+    Template.BuildNewGameStateFn = TypicalAbility_BuildGameState;
+
+    return Template;
+}
+
+static function X2AbilityTemplate MentalReadinessBuff_CI()
+{
+    local X2AbilityTemplate Template;
+    local X2Effect_PersistentStatChange StatEffect;
+
+    `CREATE_X2ABILITY_TEMPLATE(Template, 'MentalReadinessBuff_CI');
+    Template.IconImage = "img:///UILibrary_PerkIcons.UIPerk_mentalfortress";
+	Template.AbilitySourceName = 'eAbilitySource_Perk';
+	Template.eAbilityIconBehaviorHUD = EAbilityIconBehavior_NeverShow;
+	Template.Hostility = eHostility_Neutral;
+    Template.AbilityToHitCalc = default.DeadEye;
+    Template.AbilityTargetStyle = default.SelfTarget;
+    Template.AbilityTriggers.AddItem(default.UnitPostBeginPlayTrigger);
+    Template.bIsPassive = true;
+
+    StatEffect = new class'X2Effect_PersistentStatChange';
+    StatEffect.BuildPersistentEffect(1, true, false, true);
+    StatEffect.AddPersistentStatChange(eStat_Will, default.MENTAL_READINESS_VALUE); //TODO: config this
+    StatEffect.SetDisplayInfo(ePerkBuff_Passive, default.MentalReadinessFriendlyName, default.MentalReadinessFriendlyDesc, Template.IconImage, true, ,Template.AbilitySourceName);
+    Template.AddTargetEffect(StatEffect);
+
+    Template.BuildNewGameStateFn = TypicalAbility_BuildGameState;
+
+    return Template;
+}
+
+static function X2AbilityTemplate IntelligenceLeakDebuff_CI()
+{
+    local X2AbilityTemplate Template;
+    local X2Effect_PersistentStatChange StatEffect;
+
+    `CREATE_X2ABILITY_TEMPLATE(Template, 'IntelligenceLeakDebuff_CI');
+    Template.IconImage = "img:///UILibrary_PerkIcons.UIPerk_slow";
+	Template.AbilitySourceName = 'eAbilitySource_Perk';
+	Template.eAbilityIconBehaviorHUD = EAbilityIconBehavior_NeverShow;
+	Template.Hostility = eHostility_Neutral;
+    Template.AbilityToHitCalc = default.DeadEye;
+    Template.AbilityTargetStyle = default.SelfTarget;
+    Template.AbilityTriggers.AddItem(default.UnitPostBeginPlayTrigger);
+    Template.bIsPassive = true;
+
+    StatEffect = new class'X2Effect_PersistentStatChange';
+    StatEffect.BuildPersistentEffect(1, true, false, true);
+    StatEffect.AddPersistentStatChange(eStat_DetectionModifier, default.INTELLIGENCE_LEAK_DEBUFF, MODOP_Multiplication); //TODO: config this
```

Comment:
Um... I don't since you don't need to `MODOP_Multiplication` this, since the stat is already eStat_Detection**Modifier**

Reply:
wat